WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift

```diff
@@ -115,11 +97,6 @@ class MeViewController: UITableViewController {
 
         // Then we'll reload the table view model (prompting a table reload)
         handler.viewModel = tableViewModel(loggedIn)
-        //TODO: remove this after the new Me scene has shipped
```

Comment:
my bad, I should have put a TODO there too: there's a line above this, in this same method:

`let selectedIndexPath = tableView.indexPathForSelectedRow ?? IndexPath(row: 0, section: 0)`

that now needs to be removed as well